src/sidebar/components/test/Menu-test.js

```diff
@@ -74,6 +85,12 @@ describe('Menu', () => {
     assert.isTrue(isOpen(wrapper));
   });
 
+  it('gives precedence to `open` over `defaultOpen`', () => {
```

Comment:
This test is also probably poorly named, but I'm trying to make sure this interaction between props is explicit and doesn't surprise future devs.

Reply:
I think this is fine.